<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="gestion_des_avis">
       <attribute name="title">
        <string>Clients</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>631</width>
          <height>428</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Triangular</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_avis">
         <attribute name="title">
          <string>clients</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="recherche_avis">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>rechercher</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_45">
                <property name="styleSheet">
                 <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
</string>
                </property>
                <property name="text">
                 <string>trier par :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <item>
                 <property name="text">
                  <string>Nom</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>CIN</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QFormLayout" name="formLayout">
              <item row="1" column="1">
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="2" column="0">
                 <widget class="QPushButton" name="pushButton_29">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>     Modifier Client</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QPushButton" name="pushButton_37">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>     Supprimer Client</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QPushButton" name="pushButton_39">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>     Ajouter Client</string>
                  </property>
                  <property name="flat">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QPushButton" name="pushButton">
                  <property name="text">
                   <string>  Consulter Client</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QTableView" name="table_avis">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="lineWidth">
                 <number>2</number>
                </property>
                <property name="midLineWidth">
                 <number>10</number>
                </property>
                <property name="autoScrollMargin">
                 <number>16</number>
                </property>
                <property name="dragDropOverwriteMode">
                 <bool>true</bool>
                </property>
                <property name="alternatingRowColors">
                 <bool>false</bool>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::SingleSelection</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectItems</enum>
                </property>
                <property name="sortingEnabled">
                 <bool>false</bool>
                </property>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../../Downloads/res/lagoralogo.png</pixmap>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_suggestion">
       <attribute name="title">
        <string>voitures</string>
       </attribute>
       <widget class="QLabel" name="label_44">
        <property name="geometry">
         <rect>
          <x>-220</x>
          <y>350</y>
          <width>291</width>
          <height>141</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Desktop/GestClients/edit_modify_icon-icons.com_72390.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="label_52">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>480</y>
          <width>221</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Desktop/GestClients/Nouveau dossier (3)/59098-return-arrow-curve-pointing-left.png</pixmap>
        </property>
       </widget>
       <widget class="QTabWidget" name="tabWidget_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>691</width>
          <height>411</height>
         </rect>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Triangular</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_avis_3">
         <attribute name="title">
          <string>voitures</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="lineEdit_41">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>rechercher</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_54">
              <property name="styleSheet">
               <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
</string>
              </property>
              <property name="text">
               <string>trier par :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_5">
              <item>
               <property name="text">
                <string>Serie</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ID place</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="4" column="0">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Consulter Voiture</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_47">
              <property name="text">
               <string>     Supprimer Voiture</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_46">
              <property name="text">
               <string>     Ajouter Voiture</string>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>PDF</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="pushButton_36">
              <property name="text">
               <string>     Modifier Voiture</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QTableView" name="table_view_7">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="lineWidth">
               <number>2</number>
              </property>
              <property name="midLineWidth">
               <number>10</number>
              </property>
              <property name="autoScrollMargin">
               <number>16</number>
              </property>
              <property name="dragDropOverwriteMode">
               <bool>true</bool>
              </property>
              <property name="alternatingRowColors">
               <bool>false</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectItems</enum>
              </property>
              <property name="sortingEnabled">
               <bool>false</bool>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../../Downloads/res/lagoralogo.png</pixmap>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QPushButton" name="grant">
        <property name="geometry">
         <rect>
          <x>1040</x>
          <y>210</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Grant acess</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deny">
        <property name="geometry">
         <rect>
          <x>920</x>
          <y>210</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Deny</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>1050</x>
          <y>30</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ENTREE</string>
        </property>
       </widget>
       <widget class="QLabel" name="ParkingBarriere0_3">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>130</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="sources.qrc">:/new/prefix1/rc/BarrierEtat0.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="ParkingBarriere1_2">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>20</y>
          <width>61</width>
          <height>161</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="sources.qrc">:/new/prefix1/rc/BarrierEtat1.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="open">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>210</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
       <widget class="QLabel" name="ParkingBarriere1_3">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>20</y>
          <width>51</width>
          <height>171</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="sources.qrc">:/new/prefix1/rc/BarrierEtat1.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>30</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>SORTIE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="nopay">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>210</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>NO payement</string>
        </property>
       </widget>
       <widget class="QLabel" name="ParkingBarriere0_6">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>140</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="sources.qrc">:/new/prefix1/rc/BarrierEtat0.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="car">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>280</y>
          <width>151</width>
          <height>131</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="sources.qrc">:/new/prefix1/rc/car1.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_reclamation">
       <attribute name="title">
        <string>reservations</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>621</width>
          <height>411</height>
         </rect>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Triangular</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_avis_4">
         <attribute name="title">
          <string>reservations</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../../Downloads/res/lagoralogo.png</pixmap>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_38">
              <property name="text">
               <string>     Modifier Réservation</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QTableView" name="table_view_8">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="lineWidth">
               <number>2</number>
              </property>
              <property name="midLineWidth">
               <number>10</number>
              </property>
              <property name="autoScrollMargin">
               <number>16</number>
              </property>
              <property name="dragDropOverwriteMode">
               <bool>true</bool>
              </property>
              <property name="alternatingRowColors">
               <bool>false</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectItems</enum>
              </property>
              <property name="sortingEnabled">
               <bool>false</bool>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>Imprimer les Réservations</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="pushButton_9">
              <property name="text">
               <string>Export</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_48">
              <property name="text">
               <string>     Ajouter Réservation</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Consulter Réservation</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="pushButton_49">
              <property name="text">
               <string>     Supprimer Réservation</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="lineEdit_42">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>rechercher</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_55">
              <property name="styleSheet">
               <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
</string>
              </property>
              <property name="text">
               <string>trier par :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_7">
              <item>
               <property name="text">
                <string>payement</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ID</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>90</y>
          <width>231</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1189</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections/>
</ui>
